addon/index.js

```diff
@@ -1,7 +1,7 @@
 import { or, gt } from '@ember/object/computed';
 import { computed, get } from '@ember/object';
 import Mixin from '@ember/object/mixin';
-import isEqual from 'lodash/isEqual';
+import { isEqual } from 'lodash-es';
```

Comment:
Theoretically, webpack can shake out the unused named imports here because lodash-es sets `sideEffects: false` in its package.json. But I have found that to be very unreliable.

I decided to check this example, and if you change the line to 

```
import isEqual from 'lodash-es/isEqual';
```

it cuts 1.3MB from the build (before gzip)!

Reply:
Wow, is that possible because webpack doesn't add the unused modules in `lodash-es` to the build (which it cannot reliably do if I use named imports from the main package name)?

Reply:
Correct. 

Lodash itself is already structured into a ton of small files, which allows you to pull on the specific entrypoints (like `lodash-es/isEqual`) that you need, and those will pull in the other ones that they need, but not others.